src/index.html

```diff
@@ -7,8 +7,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link is-active" href="#apple">
+            apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#samsung"
+            >
+            samsung
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
add an empty line between _multiline_ siblings (code styling)
```suggestion
          </li>
          
          <li class="nav-item">
```